nucliadb_vectors2/src/data_point_provider/reader.rs

```diff
@@ -175,25 +179,28 @@ impl Reader {
         let mut new_dimension = self.dimension;
         let mut new_number_of_embeddings = 0;
         let mut new_data_points = Vec::new();
+        let mut new_open_data_points = Vec::new();
```

Comment:
I think we might want to try to avoid closing & reopening data points during the update to avoid paying the `munmap` cost unnecessarily. At the very least, I'd add a histogram metric for `update()` time so we can see if it's becoming a problem or not.

Reply:
Sure! Will add this to the PR

Reply:
Done!